cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -59,6 +59,7 @@
       cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'fetch_external_resources'), notify:'dev+crontab@code.org'
       cronjob at:'30 17 * * *', do:deploy_dir('bin', 'cron', 'update_dotd')
       cronjob at:'0 13 * * *', do:deploy_dir('tools', 'scripts', 'brokenLinkChecker', 'brokenLinkChecker.js'), notify:'dev+crontab@code.org'
+      cronjob at:'30 15 * * 1-5', do:deploy_dir('bin', 'cron', 'deploy_to_test')
```

Comment:
It seems excessive to add a line for every row. At the same time (beyond on the other side for regex), I do think a comment at the top with a useful link is appropriate. Let me do so in another PR.